dummy-api/profile/overview.ts

```diff
@@ -0,0 +1,450 @@
+import PlayerTshirtIcon from '@/icons/PlayerTShirtPngIcon.png'
+import PlayerUpIcon from '@/icons/PlayerUpPngIcon.png'
+import SecondLeagueIcon from '@/icons/SecondLeague.png'
+import ThirdLeagueIcon from '@/icons/ThirdLeague.png'
+
+import { IProfileCareerData } from './career'
+import { IProfileMatchesSeason } from './matches'
+
+export interface IProfileOverviewData {
+  inShort: {
+    title: string
+    icon: string
+  }[]
+  transfers: {
+    id: number
+    from: {
+      logo: string
+      name: string
+    }
+    to: {
+      logo: string
+      name: string
+    }
+    type: string
+    date: string
+  }[]
+  similarPlayers: {
+    name: string
+    image: string
+    score: number
+    premium: boolean
+    carrer: string
+    age: number
+    trend: 'up' | 'down'
+  }[]
```

Comment:
please don't ignore stuff from previous review
We do that because backend shouldn't worry about how we format data for display
```suggestion
  similarPlayers: {
    name: string
    image: string
    score: number
    premium: boolean
    age: number
    scoreTrend: 'up' | 'down'
    position: string
    team: string
    competition: string
  }[]
```
you'll have to adjust component as well